src/FakeItEasy/StrictFakeOptions.cs

```diff
@@ -32,6 +32,11 @@ public enum StrictFakeOptions
         /// <summary>
         /// Calls to all methods inherited from <see cref="object"/> are allowed, and behave as if the fake weren't strict.
         /// </summary>
-        AllowObjectMethods = AllowEquals | AllowGetHashCode | AllowToString
+        AllowObjectMethods = AllowEquals | AllowGetHashCode | AllowToString,
+
+        /// <summary>
+        /// Events of the fake are managed automatically (subscription, unsubscription and raising).
+        /// </summary>
+        ManageEvents = 8

```

Comment:
I'm not very happy with this name. Maybe it should be `AllowEvents`, like the other options, but I wanted to convey the fact that it had a special behavior. But now that I think of it, that special behavior has nothing to do with the fake's strictness...
Let me know if you prefer a different name.

Reply:
@blairconrad I think you missed my last comment, since the conversation was resolved. I unresolved to make it more visible.

Reply:
I typed out a whole comment! It's lost. Sorry. 

I don't mind `ManageEvents`, since we don't need consistency in order to locate the option. However, I think `AllowEvents` is as good. There's (in my mind) no special behaviour here. Just like `AllowEquals` or the others, it's enabling the behaviour we'd have as if the fake wasn't strict.
I guess if I had to pick, I'd probably keep "Allow", but don't mind if you keep Manage.

Reply:
OK, I'll rename it then.